src/__tests__/scenes/CreateWalletSelectCryptoScene.test.tsx

```diff
@@ -46,21 +47,27 @@ describe('CreateWalletSelectCrypto', () => {
               }
             }
           }
+        },
+        currencyWallets: {
+          '123SomeWalletId': {
+            currencyInfo: {
+              currencyCode: 'ETH',
+              displayName: 'Ethereum',
+              pluginId: 'ethereum'
+            }
+          }
         }
       }
     }
   }
   const store = createStore(rootReducer, mockState)
 
   it('should render with loading props', () => {
-    const renderer = createRenderer()
-
     const props: any = {
       navigation: fakeNavigation,
-      account: () => fakeUser,
```

Comment:
I really don't like this `const props = { ... }` followed by `<TestComponent {...props} />`. We should just put the props on the component, like normal. That way TypeScript will check them individually and warn you about extras or missing ones. However, as long as your tests work (which they seem to be), we don't need to fix this right away.

Reply:
I went ahead and did this and actually caught a bug in the cryptorow test where the icon wasn't appearing in the snapshot.